client/a8c-for-agencies/sections/referrals/referral-details/components/assigned-to.tsx

```diff
@@ -78,15 +88,103 @@ const AssignedTo = ( { purchase, handleAssignToSite, data, isFetching }: Props )
 		);
 	}
 
+	let tooltip = '';
+
+	let cancellationInfo = null;
+
+	// If the purchase has a subscription we can get the information from the object.
+	if ( purchase.subscription && ! purchase.subscription.is_refundable ) {
+		cancellationInfo = (
+			<>
+				<p>
+					{ translate(
+						'This product was cancelled, but it will remain active until {{b}}%(expiryDate)s{{/b}}. After that, it will not renew.',
+						{
+							args: {
+								expiryDate: isFetchingProductInfo ? '...' : expiryDate || 'N/A',
+							},
+							components: {
+								b: <b />,
+							},
+							comment: '%(expiryDate)s is the date when the product will expire.',
+						}
+					) }
+				</p>
+				<p>
+					{ translate( '{{link}}Learn more about cancelations ↗{{/link}}', {
+						components: {
+							link: (
+								<a
+									href={ localizeUrl(
+										'https://wordpress.com/support/manage-purchases/cancel-a-purchase/'
+									) }
+									target="_blank"
+									rel="noreferrer noopener"
+								>
+									{  }
+								</a>
+							),
+						},
+					} ) }
+				</p>
+			</>
+		);
+	} else if ( purchase.subscription && purchase.subscription.is_refundable ) {
+		cancellationInfo = <p>{ translate( 'This product was cancelled, and was refunded.' ) }</p>;
```

Comment:
I'm not quite sure how we'd know for sure if something was really refunded and trying to guess based on some of these properties seems risky. I don't see anything in the subscription or ownership record that points to something having been refunded. I'm sure it's possible but we'd probably need to get it from the receipts.

IMO, I think it's fine to remove this whole tooltip since if something was truly canceled it'll already have the `Canceled` label.